sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -666,7 +695,8 @@ Exceptions:
 |
 [source, c++]
 ----
-bool begin_recording(const std::vector<queue>& recordingQueues)
+bool begin_recording(const std::vector<queue>& recordingQueues, 
+const property_list& propList = {})
```

Comment:
I should at least be consistent across PRs with my nitpicking

```suggestion
bool begin_recording(const std::vector<queue>& recordingQueues, 
                     const property_list& propList = {})
```

Reply:
Fixed!